Jenkinsfile

```diff
@@ -23,71 +23,31 @@ node('docker') {
     stage('Build Extension') {
         img.inside() {
             withCache {
-                sh 'rm -rf build'
-                if (params.WITH_CLIQZ_MASTER) {
-                    sh 'npm install --save https://s3.amazonaws.com/cdncliqz/update/edge/ghostery/master/latest.tgz'
-                }
-                // make browser-core noisy
-                sh 'sed -i \'s/global.__DEV__/true/1\' node_modules/browser-core/build/core/console.js'
-                withGithubCredentials {
-                    sh 'moab makezip'
+                withEnv(["NO_LINT=true"]) {
```

Comment:
@christophertino is this step always required? 
Will merge this PR to keep the train moving but we probably need to double check this again before merging back to your branches (this and probably many more :) )